default.yaml

```diff
@@ -86,6 +99,16 @@ proxies:
       # a ZooKeeper leader election in your setup.
       shutdown_timeout: 30s
 
+      # How to assign incoming messages to a Kafka partition. Defaults to using
+      # a hash of the specified message key, or random if the key is
+      # unspecified.
+      partitioner: hash
```

Comment:
Oops, I missed that here there is no list of possible values and their meanings similar to [require_acks](https://github.com/mailgun/kafka-pixy/blob/master/default.yaml#L74-L82) parameter

Reply:
Sure, fixed.